packages/app/src/explorer/EntityItem.tsx

```diff
@@ -57,11 +58,12 @@ function EntityItem(props: Props) {
       >
         <Icon entity={entity} isExpanded={isExpanded} />
         <span className={styles.name}>{entity.name}</span>
-        {isNxGroup(entity) && (
-          <span className={styles.nx} aria-label=" (NeXus group)">
-            NX
-          </span>
-        )}
+
+        <ErrorBoundary fallback={<> </>}>
```

Comment:
`ErrorBoundary` doesn't accept `null` as fallback. The space is to avoid an ESLint warning.